app/components/blacklight/facet_item_component.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
```

Comment:
This file is almost entirely copied from the existing `FacetItemComponent` in Blacklight, with a few additions noted in comments below. 

If there is a better way to do this than overriding the component file (subclassing the component, maybe?), please let me know. It seems not great to me to have to override everything in this component just to add some small functionality, so maybe I'm missing something here! 🤓